src/components/Filters/StatefulFilters.tsx

```diff
@@ -70,42 +79,24 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
   }
 
   componentDidUpdate(prevProps: StatefulFiltersProps, prevState: StatefulFiltersState, snapshot: any) {
-    const filtersExists = (prevProps.initialActiveFilters || []).every(prevFilter => {
```

Comment:
Where is this logic replaced ?
I miss the trace with the whole refactoring, where is checked duplication on filters ?

Reply:
If I understand correctly this code, it's more about checking if there is any change in filters (between prev props and new props) rather than anything about duplicate filters.
The logic is a bit changed and now `filtersMatchURL` plays an similar role.
I'm myself a bit scared about not missing a hidden corner case when I look at the pieces of code removed :-s So manual tests are quite important here.

I just tested that duplicated filters are correctly handled, and it's ok. In StatefulFilters.tsx it's still managed, cf function `duplicatesFilter`  which is still there.

Reply:
My preliminar smoke test looks ok, nothing seems broken so far.
Anyway, @hhovsepy can also help to review this PR, at least to check nothing is missing.